features/title-attr.yml

```
@@ -0,0 +1,6 @@
+name: Title
+description: The `title` global attribute and property represents the title of an element, with the attribute value typically displayed in a 'tooltip' popup when the mouse is over the node.
```

Should we mention something about the accessibility issues of this attribute?

```suggestion
description: The `title` global attribute sets information about the element it belongs to. The attribute value is typically displayed as a tooltip that appears on mouse over, which can cause accessibility issues for touch-only, keyboard-only, or assistive technology users.
```